### ISSUE TITLE
Cellular HTML5: can "don't rotate" be default for sprites?

### ISSUE THREAD

**Issue by mciotola** [aidanlane/snapapps#61]
Cellular HTML5: 

Can "don't rotate" be the default for sprites? This sounds trivial, but most of our examples do not use rotation. I keep forgetting to click "don't rotate", so I have to often re-do work. I expect the same for most of our users.
![cellular_html5_sprite_rotation_choices](https://cloud.githubusercontent.com/assets/714256/7690563/7015e6dc-fd69-11e4-92d7-0ce69c0c46a5.png)

**Comment by aidanlane** [comment]
What was the default in the desktop app?

@berndmeyer, any thoughts?

**Comment by mciotola** [comment]
The default was "can rotate", so this would be a change from the desktop
version. It works for ants and bacteria, but not so well for bushfires,
palm trees, cows, people or viruses (which are smiley faces sometimes with
hats).

**Comment by aidanlane** [comment]
"Can rotate" is also the default in scribble too, right?

**Comment by mciotola** [comment]
Yes, although the sprite in Scribble is typically a pen or pencil, rather
than, say a cow or person.

It the default stays on "can rotate", it is not a problem, merely a minor
annoyance.

**Comment by aidanlane** [comment]
Cool- I'll let @berndmeyer make the call... :)

**Comment by berndmeyer** [comment]
I handball that to Mark - you're closer to how the examples work. Happy to
go with whatever you see fit!

**Comment by mciotola** [comment]
Matt, do you have any insights on whether the default for sprites being
"can rotate" versus don't rotate?
